### PR TITLE
Expose everywhere

### PR DESCRIPTION
**Pull request by Ms2ger**
Ref: https://github.com/tc39/proposal-shadowrealm/issues/331

- [x] At least two implementers are interested (and none opposed):
   * Positive initial signals from Google, Apple, Mozilla on the general direction.
- [x] [Tests](https://github.com/web-platform-tests/wpt) are written and can be reviewed and commented upon at:
   * https://github.com/web-platform-tests/wpt/pull/34361
- [x] [Implementation bugs](https://github.com/whatwg/meta/blob/main/MAINTAINERS.md#handling-pull-requests) are filed:
   * Chrome: https://bugs.chromium.org/p/chromium/issues/detail?id=1281880
   * Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1747059
   * Safari: https://bugs.webkit.org/show_bug.cgi?id=234554

(See [WHATWG Working Mode: Changes](https://whatwg.org/working-mode#changes) for more details.)

***
<a href="https://whatpr.org/console/209.html" title="Last updated on Jun 14, 2022, 8:13 AM UTC (334b6f3)">Preview</a> | <a href="https://whatpr.org/console/209/21f55a4...334b6f3.html" title="Last updated on Jun 14, 2022, 8:13 AM UTC (334b6f3)">Diff</a>

### CODE REVIEW

**Comment by terinjokes** [comment]
LGTM, though I'm wondering if we need to specify the side effects that happen in some of these other environments.

**Comment by Ms2ger** [comment]
On a first glance, I'm not seeing anything that obviously needs adjusting, but let me know if you disagree.

**Comment by annevk** [comment]
@terinjokes @domfarolino can you sort out the CI failure? Seems like an image might need width/height attributes?

**Review by domfarolino — APPROVED** [review]
Regarding tests, you have:

> TBD but quite trivial

This PR LGTM, but do you plan on adding tests?

**Comment by Ms2ger** [comment]
@domfarolino For tests, I'll make sure to add an idlharness test as soon as the IDL changes propagate into the wpt repo after this PR is merged. I don't think there's much else I can test - maybe a manual test that the messages show up?

**Comment by domfarolino** [comment]
A manual test would be nice I think

**Comment by Ms2ger** [comment]
Tests are up at https://github.com/web-platform-tests/wpt/pull/34361

**Comment by domfarolino** [comment]
This and the tests LGTM. GitHub is complaining about the branch not being up to date, so if you could fix that then I think we'd be good to go.

**Comment by Ms2ger** [comment]
Done!

**Comment by domfarolino** [comment]
Great, feel free to merge the tests whenever and I'll go ahead and merge this.

**Comment by Ms2ger** [comment]
Merged the wpt pr, thank you

**Pull request merged**
